tensorflow/stream_executor/blas.h

```diff
@@ -101,13 +101,72 @@ enum class ComputationType {
   kI32,         // 32-bit integer
   kComplexF32,  // Complex number comprised of two f32s.
   kComplexF64,  // Complex number comprised of two f64s.
+  // The below values are only supported for BlasLt routines (both real and
+  // complex).
+  kF32FastTF32, // 32-bit floating-point with reduced (>=10-bit) mantissa
```

Comment:
Can they be "kTF32" and "kBF16"?

Reply:
The naming is a bit tricky here. Both use F32 accumulation, with the difference being the mantissa size that the input is rounded to (I've added a comment describing this behavior in more detail). For this reason I'm not sure that TF32/BF16 would be sufficient, though I'm open to changing them if you think it would be better.

Reply:
Done